common/presenters/move-to-card-component.js

```diff
@@ -29,6 +30,24 @@ function moveToCardComponent({
 
     const moveMetaItems = []
 
+    if (showIdentifierNumber && profile?.person) {
```

Comment:
So, after writing the comment above I noticed that we actually have a presenter to handle rendering a [profile as a card](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/0adb469d7a6508dfb7e79f3b394bb4aab1b9834a/common/presenters/profile-to-card-component.js#L43). This is also what's actually used to render the person on the search results page:

<kbd>![image](https://user-images.githubusercontent.com/3327997/117273487-1784d800-ae54-11eb-8189-2969b8324ecc.png)</kbd>

Which lead me to think that we should actually move this code to the profile presenter and then we would also be able to see the identifier on the search results too. @ElizabethGoddard should we show it there as well?

You can do away with the flag on this presenter and just pass `locationType` to `profileToCardComponent` and re-use that logic there.

The more I think about it, the identifier is part of the profile and not the move so I do think this code should be located in the other presenter.

Reply:
Hey @teneightfive and @Thource if it makes sense to do the code this way then yes we might as well show it on the search. It might not add that much value now but it may one day if we were to increase the scope to search (e.g. by name and then get a listing, the Prison/PNC number would distinguish the results).